.circleci/config.yml

```diff
@@ -274,7 +282,7 @@ jobs:
           command: (cd client && npm run ci_serve)
           background: true
       - run: (cd client && npm run ci_route_load_tests)
```

Comment:
The old route load tests don't work in the current docker image, but that's fine. Cypress seems to be working out so probably about time to go around deleting all the old route load test stuff.

In theory it'd be best to run the old and new tests alongside eachother for a bit, but the old tests were flaky anyway so that would just be noisy. Better to just re-test the cypress equivalents on their own (e.g. go around introducing different types of error-throwing bugs in to different routes and re-running the tests to make sure they're properly caught).

Reply:
Yeah it clicked that the old tests weren't going to work as soon as I pushed, forgot about the image🤦‍♂️